extension/js/common/ui/key-import-ui.ts

```diff
@@ -356,7 +356,19 @@ export class KeyImportUi {
       const keyinfos = await KeyStore.get(acctEmail);
       const privateKeysIds = keyinfos.map(ki => ki.fingerprints[0]);
       if (privateKeysIds.includes(k.id)) {
-        throw new UserAlert('This is one of your current keys, try another one.');
+        const key = keyinfos.find(ki => ki.id === k.id);
+        /* eslint-disable @typescript-eslint/no-non-null-assertion */
+        const existingKey = await KeyUtil.parse(key!.public);
```

Comment:
`eslint-disable` here will disable `no-non-null-assertion` rule for all lines until the end of the file. `eslint-disable-next-line` will be more correct to use here.

but it'll be even better to add additional check than use `eslint-disable`:
```ts
const key = keyinfos.find(ki => ki.id === k.id);
if (key) {
  const existingKey = await KeyUtil.parse(key.public);
  ...
```

Reply:
Thanks, I'll be correcting this.